projects/plugins/boost/app/modules/optimizations/minify/class-css.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Modules\Optimizations\Minify;
+
+use Automattic\Jetpack_Boost\Contracts\Pluggable;
+
+class Css extends Minify implements Pluggable {
+
+	public static $default_excludes = array( 'admin-bar', 'dashicons' );
+
+	public static function get_slug() {
+		return 'minify_css';
+	}
+
+	public function init_concatenate() {
+		if ( is_admin() ) {
+			return;
+		}
+
+		jetpack_boost_init_filesystem();
+
+		global $wp_styles;
+
+		// phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+		$wp_styles                         = new Concatenate_CSS( $wp_styles );
+		$wp_styles->allow_gzip_compression = true; // @todo - used constant ALLOW_GZIP_COMPRESSION = true if not defined.
```

Comment:
`@TODO` items should be turned into tickets before a PR is merged.

Reply:
Opened an issue listing all todos https://github.com/Automattic/jetpack/issues/30080.